packages/arb-token-bridge-ui/tests/e2e/specs/withdrawETH.cy.ts

```diff
@@ -75,6 +92,63 @@ describe('Withdraw ETH', () => {
               })
           })
       })
+
+      it('should claim funds', { defaultCommandTimeout: 200_000 }, () => {
+        // increase the timeout for this test as claim button can take ~(20 blocks *10 blocks/sec) to activate
+
+        cy.login({ networkType: 'parentChain' }) // login to L1 to claim the funds (otherwise would need to change network after clicking on claim)
+
+        cy.findByLabelText('Open Transaction History')
+          .should('be.visible')
+          .click()
+
+        cy.waitUntil(
+          () =>
+            cy
+              .findByLabelText(
+                `Claim ${formatAmount(ETHToWithdraw, {
+                  symbol: 'ETH'
+                })}`
+              )
+              .should('be.visible')
+              .should('not.be.disabled'),
+          {
+            errorMsg: 'Claim Transaction button is not visible or enabled',
+            timeout: 200_000,
+            interval: 1000
+          }
+        ).then(() => {
+          cy.findByLabelText(
+            `Claim ${formatAmount(ETHToWithdraw, {
+              symbol: 'ETH'
+            })}`
+          )
+            .click()
+            .then(() => {
+              cy.confirmMetamaskTransaction().then(() => {
+                cy.findByLabelText('show settled transactions')
+                  .should('be.visible')
+                  .click()
+
+                cy.findAllByText(
+                  `${formatAmount(ETHToWithdraw, {
+                    symbol: 'ETH'
+                  })}`
+                )
+                  .first()
+                  .should('be.visible')
+
+                cy.findByLabelText('Close side panel').click()
+
+                // the balance on the destination chain should not be the same as before
+                cy.findByLabelText('ETH balance amount on parentChain')
```

Comment:
same here, can do `l1EthBal + ETHToWithdraw`

Reply:
Addressed in previous reply